packages/react-components/src/components/FileCard.tsx

```diff
@@ -49,13 +50,13 @@ export const FileCard = (props: FileCardProps): JSX.Element => {
   const { fileName, fileExtension, progress, actionIcon } = props;
   const theme = useTheme();
 
-  const showProgressIndicator = (): boolean => progress !== undefined && progress < 1;
+  const showProgressIndicator = (): boolean => progress !== undefined && progress > 0 && progress < 1;
```

Comment:
nit: Perf papercut warning: This can be a file-level private function. That will avoid creating a new closure on every render. Coming from a static languages background, I balk at how freely we create new closures, but I don't have a real understandig of how costly closures are in JavaScript.

Reply:
Makes sense. Updated the code.